crates/commons/Cargo.toml

```diff
@@ -17,3 +17,6 @@ time = { version = "0.3", features = ["serde", "std"] }
 tokio-tungstenite-wasm = { version = "0.3.0" }
 trade = { path = "../trade" }
 uuid = { version = "1.3.0", features = ["v4", "serde"] }
+
+[dev-dependencies]
+secp256k1 = { version = "0.27.0", features = ["serde", "rand", "global-context"] }
```

Comment:
nit: this shouldn't be needed if we use the `verify_ecdsa` function to verify signatures.

Reply:
this is needed for generating the secret keys and a public key from it.